.github/workflows/publish_release.yml

```diff
@@ -18,13 +18,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      # Needed to get access to publish-release action definition
-      - name: Check out the released repo
-        uses: actions/checkout@v3
-        with:
-          # Fetch whole repo to get access to tags to read current package
-          # version.
-          fetch-depth: 0
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Did you mean to remove this? If so, will releases work?

Reply:
Good catch, bringing it back